cmd/ipfs-cluster-follow/main.go

```diff
@@ -271,6 +278,25 @@ as obtained from the internal state on disk.
 `,
 				Action: listCmd,
 			},
+			{
+				Name:      "stop",
+				Usage:     "stop to follow the cluster and unpin everything",
+				ArgsUsage: "",
+				Description: fmt.Sprintf(`
+
+This command stops following %s and leaves %s. It also removes any
+persisted consensus data, including the current pinset (state).
+
+The next start of this follower will be like a first start to all effects.
+`, clusterName, clusterName),
+				Action: stopCmd,
+				Flags: []cli.Flag{
+					&cli.BoolFlag{
+						Name:  "cleanup",
+						Usage: fmt.Sprintf("delete files in ~/.ipfs-cluster-follow/%s", clusterName),
```

Comment:
You would have to use `buildPaths()` or join `c.String("config")` with `clusterName` to obtain the right path. It may not be `~/.ipfs-cluster-follow` if it was customized.